uber/site_sections/summary.py

```diff
@@ -175,12 +175,115 @@ def food_eligible(self, session):
         }
         return render('summary/food_eligible.xml', {'attendees': eligible})
 
+    def csv_import(self, message='', all_instances=None, form_link=None):
```

Comment:
I'm kind of scared about this being in the section which (for MAGFest) has the least restrictive access level.  I'm inclined to think that this should be limited to either only people with ``c.ACCOUNTS`` access.